frontend/src/components/Editor/Editor.tsx

```diff
@@ -89,13 +98,17 @@ const Editor = ({name, text, endpoint}: {
     }, []);
 
     return (
-        <Card className={style.content} title={name}>
+        <Card className={style.content} style={{marginTop: '64px',}} title={name}>
             <div className={style.editorOptionsPanel}>
                 <LanguageSelector language={language} onSelect={onLanguageChange}/>
-                <Button type="primary" onClick={onSave}>Save</Button>
+                <div style={{display: 'flex', gap: '8px'}}>
+                    <Button type="default" onClick={() => navigate(-1)}>Back</Button> {/* Кнопка Back */}
```

Comment:
```suggestion
                    <Button type="default" onClick={() => navigate(-1)}>Back</Button>
```

Reply:
Kto na przycisk mówi knolka to ja nwm xd